lib/src/described_feature_overlay.dart

```diff
@@ -143,10 +147,23 @@ class _DescribedFeatureOverlayState extends State<DescribedFeatureOverlay>
     _openController.dispose();
     _completeController.dispose();
     _dismissController.dispose();
-    _pulseController?.dispose();
+    _pulseController.dispose();
     super.dispose();
   }
 
+  @override
+  void didUpdateWidget(DescribedFeatureOverlay oldWidget) {
+    if (oldWidget.enablePulsingAnimation != widget.enablePulsingAnimation) {
+      if (widget.enablePulsingAnimation)
+        _pulseController.forward(from: 0);
+      else {
+        _pulseController.stop();
```

Comment:
@axel-op To make it look better?

I thought about this initially, but I found three reasons why I did not do it:

 1. It does not look bad to stop the pulsing abruptly. For hot reload, it would be somewhat annoying if it did not stop immediately. If one were to stop it programmatically, they would probably have some other change in UI accompany it.

 1. This is also how the pulsing animation is stopped when closing the overlay.

 1. It adds unnecessary weight and complexity. I cannot imagine someone using it programmatically anyway. Maybe someone has a scenario for it, but it is very helpful for debugging.

Reply:
You're right, except that for your number 2, we should find a way to change that (in the dismiss method of the overlay)